integr8ly/installation
# remove unused templates from ups

## Additional Information

This change removes unused templates from the UPS role. 

These templates are currently handled by the UPS operator

## Verification Steps
As the verifier of the PR the following process should be done:
- Install RHMI from branch
- Ensure successful install
- Run `oc get prometheusrule -n mobile-unifiedpush -o yaml` 
- Ensure rules are as expected
- Run `oc get grafanadashboard -n mobile-unifiedpush -o yaml`
- Ensure dashboards are as expected

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 


## Checklist



Is an upgrade task required and are there additional steps needed to test this?
